src/model_tuner/bootstrapper.py

```diff
@@ -7,6 +7,68 @@
 from random import seed, randint
 
 
+def check_input_type(x):
+    """Method to check input type pandas Series or numpy.
+    Sort index if pandas for sampling efficiency.
+    """
+    # if y is a numpy array cast it to a dataframe
+    if isinstance(x, np.ndarray):
+        x = pd.DataFrame(x)
+    elif isinstance(x, pd.Series):
+        x = x.reset_index(drop=True)  # have to reset index
+    elif isinstance(x, pd.DataFrame):
+        x = x.reset_index(drop=True)  # have to reset index
+    else:
+        raise ValueError("Only numpy or panndas types supported.")
+    return x
+
+
+def sampling_method(
+    y,
+    n_samples,
+    stratify,
+    balance,
+):
+    """Method to resample a dataframe balanced, stratified, or none.
+
+    Returns resampled y.
+    """
+    if balance:
```

Comment:
We need to test whether this works for regression e.g. we can't balance classes with regression so if balance is true then get rid then set balance to False